src/lib/components/ui/Button/Button.jsx

```diff
@@ -178,6 +127,10 @@ Button.propTypes = {
    * Function to call when the button is clicked.
    */
   clickHandler: PropTypes.func,
+  /**
+   * [Color variant](/foundation/colors#component-colors) to clarify importance and meaning of the button.
+   */
+  color: PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'help', 'info', 'note', 'light', 'dark']),
```

Comment:
It would be better to store this array into constant that will be shared across all components. (It is applicable for all components)

Reply:
> This is unfortunately still blocked by #86.

Reply:
🤷🏻‍♂️